.travis.yml

```diff
@@ -40,6 +40,10 @@ addons:
       - gcc-multilib
       - lib32gcc-6-dev
 
+before_script:
+  - wget http://releases.linaro.org/components/toolchain/binaries/6.3-2017.02/aarch64-linux-gnu/gcc-linaro-6.3.1-2017.02-x86_64_aarch64-linux-gnu.tar.xz
```

Comment:
Thank you a lot for your work.
It added less than 2 minutes to travis check which is pretty impressive.

I wonder if it is possible to use http://releases.linaro.org/components/toolchain/binaries/latest/ link and hardcoded version.